metricflow/query/group_by_item/candidate_push_down/group_by_item_candidate.py

```diff
@@ -49,7 +49,7 @@ def intersection(
         where the new candidate set was created.
         """
         specs_as_sets = tuple(set(candidate_set.specs) for candidate_set in candidate_sets)
-        common_specs = set.intersection(*specs_as_sets)
+        common_specs = set.intersection(*specs_as_sets) if specs_as_sets else {}
```

Comment:
Might need to be `set()` as `{}` seems to evaluate to a dict when I tried it out.

Reply:
le sigh